<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1222</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>741</width>
     <height>461</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>foot.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="left0Label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="left1Label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="left2Label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>310</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="left3Label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="left4Label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>430</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="right4Label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>440</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="right3Label">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>390</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="right2Label">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="right1Label">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="right0Label">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>270</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="leftAvgLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>600</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No Data</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="rightAvgLabel">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>600</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No Data</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>600</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AVG : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>600</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AVG : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
